sunpy/__init__.py

```diff
@@ -48,4 +48,23 @@ class UnsupportedPythonError(Exception):
     # Load user configuration
     config = load_config()
 
+    import logging
+
+    # Use the root logger as a dummy log before initializing Astropy's logger
+    log = logging.getLogger()
+
+    from astropy.logger import _init_log
+
+    #  read and translate settings for Astropy's logger
+    if config.has_section('logger'):
+        from astropy.logger import Conf as LoggerConf
+        conf = LoggerConf()
+        option_list = ['log_level', 'use_color', 'log_warnings', 'log_exceptions', 'log_to_file', 'log_file_path',
```

Comment:
Can we read these options out of the config file rather than listing them here?

Reply:
These are specific attributes to the LoggerConf object that astropy defined. I guess I was trying to protect from an error if the user accidentally uses an incorrect config option.